metricbeat/module/windows/perfmon/reader.go

```diff
@@ -74,9 +75,10 @@ func NewReader(config Config) (*Reader, error) {
 				return nil, errors.Wrapf(err, `failed to expand counter (query="%v")`, counter.Query)
 			}
 		}
-		if childQueries == nil || len(childQueries) == 0 {
+		// check if the pdhexpandcounterpath/pdhexpandwildcardpath functions have expanded the counter successfully.
+		if childQueries == nil || len(childQueries) == 0 || (len(childQueries) == 1 && strings.Contains(childQueries[0], "*")) {
```

Comment:
Is it not possible that a wildcard expands to a single path?

Reply:
@jsoriano , yes that should be possible but the path should not still contain the wildcard "*", this is why the condition  `(len(childQueries) == 1 && strings.Contains(childQueries[0], "*")`. Am I missing something here?

Reply:
Oh ok, sorry, I misunderstood, all good.